sdk/python/src/freewayai/llm_system.py

```diff
@@ -1,7 +1,7 @@
-from .prompt_formatted import Prompt_Formatted
+from .prompt_template import PromptTemplate
 from .database_query_template_format import read_and_format_query
 from .utilities import read_prompt_from_system_and_id
-from .prompt_formatted import Prompt_Formatted
+from .prompt_template import PromptTemplate
```

Comment:
We probably didn't catch this before and you did a find/replace but you're importing twice here. Reminds me we should add flake8 and black 😬